free-1.md

```diff
@@ -57,12 +68,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The "words" of this language is simply every key on the keyboard, and since there are no limitations on which the sequences that these keys are used, there aren't any language design decisions in the traditional sense. 
+
+If I were to choose the option of translating text into music and sound, it would be very easy for the sound to devolve into some hodgepodge of unappealing sounds. So there would be the interesting problem of how to structure it in a way that would avoid this. If I were to pursue the real-time option, then there would be design questions of which keys should produce which notes (the most accessible keys producing the most standard sounds?), and how to implement the saving and editing the layers of sound (using the arrow keys, perhaps). 
```

Comment:
I see this answers my earlier question of what proportion of the available space of possible sounds this language will allow the user to explore.